packages/wdio-utils/tests/utils.test.ts

```diff
@@ -203,6 +211,7 @@ describe('utils:canAccess', () => {
         expect(canAccess('/foobar')).toBe(true)
         expect(fs.accessSync).toBeCalledWith('/foobar')
 
+        // @ts-ignore
```

Comment:
is it possible to do something like this `(fs.accessSync as jest.Mock)`?

Reply:
Nope, unfortunately not which is why I added it there like that. Happy to revisit all `@ts-config` and refine them once we moved over.

Reply:
Yes, you can do `jest.Mock<any, any>` if it allows that where the first any is what it put in (believe) and the 2nd any is what is returned if I am correct